doc/htmldoc/installation/index.rst

```diff
@@ -3,40 +3,69 @@
 Install NEST
 ============
 
-.. grid:: 1 1 2 2
 
-   .. grid-item-card::  |user| Install pre-built NEST package
-       :class-title: sd-d-flex-row sd-align-minor-center
+.. grid:: 2
+   :gutter: 1
 
-       I'm a user who wants to :ref:`install NEST on my computer <user_install>`
+   .. grid-item-card:: |user|  Install a pre-built NEST package using
+      :class-title: sd-d-flex-row sd-align-minor-center
+      :columns: 4
 
+      * :ref:`Docker (cross-platform) <docker>`
+        (Includes NESTML)
+      * :ref:`conda-forge (Linux/macOS) <conda_forge_install>`
+      * :ref:`Ubuntu PPA (Linux) <ubuntu_install>`
+      * :ref:`Homebrew (macOS) <macos_install>`
+      * :ref:`Options for Windows users <windows_install>`
 
-   .. grid-item-card:: |teacher| Install NEST for a class or workshop
-       :class-title: sd-d-flex-row sd-align-minor-center
 
-       I'm a lecturer who wants to :ref:`use NEST to teach <lecturer>`
+   .. grid-item-card:: |dev| Source install
+      :class-title: sd-d-flex-row sd-align-minor-center
+      :columns: 4
 
+      If you want to do development with NEST
 
-.. grid:: 1 1 2 2
+      * :ref:`dev_install`
 
-    .. grid-item-card:: |admin| Install NEST for supercomputers and clusters
-       :class-title: sd-d-flex-row sd-align-minor-center
+   .. grid-item-card:: |launch| Try NEST installation-free
+      :class-title: sd-d-flex-row sd-align-minor-center
+      :columns: 4
 
-       I'm an admin or user who wants to :ref:`run NEST on HPC <admin_install>`
+      With the JupyterHub service from EBRAINS you can
+      try out NEST (and many other tools) directly in your browser.
 
-    .. grid-item-card:: |dev| Install NEST from source
-       :class-title: sd-d-flex-row sd-align-minor-center
+      .. rst-class:: imgbutton
 
```

Comment:
Aren't the public NEST desktop installations also options for using NEST installation-free?
Hifis: https://nest-desktop.helmholtz-cloud.desy.de
Ebrains: https://nest-desktop.apps.hbp.eu

Reply:
do we want to directly link to NEST desktop applications? I was thinking of directing users to the NEST desktop documentation, and let users discover the available ways of using NEST desktop through their site

Reply:
It's not necessary to link directly, but these things can be mentioned. But that is not a stop for the PR.